### PR TITLE
chore(grid): update prop descriptions

### PR DESCRIPTION
**Pull request by Mikaelia**
## Description

Updates grid package prop docs

### CODE REVIEW

File: packages/grid/src/elements/Col.tsx [hunk 1]
```diff
@@ -19,80 +19,79 @@ import { StyledCol } from '../styled';
...
 export interface IColProps extends HTMLAttributes<HTMLDivElement> {
-  /**
-   * Determine the size, relative to the total number of `columns` in the grid,
-   * for all screen sizes
-   */
+  /** Sets the total number of grid `columns` that the col spans on all screen sizes */
   size?: GRID_NUMBER;
-  /** Determine the size for extra-small screen sizes */
+  /** Defines the col size on extra-small screens.
+   *  See [breakpoints](https://garden.zendesk.com/components/theme-object#default_theme).
```
Comment (jzempel) [inline packages/grid/src/elements/Col.tsx:25]: Let's use the `website`-local URL here, and maybe change to `/components/theme-object#breakpoints`

File: packages/grid/src/elements/Row.tsx [hunk 2]
```diff
@@ -20,49 +20,49 @@ import { StyledRow } from '../styled';
 
 export interface IRowProps extends HTMLAttributes<HTMLDivElement> {
   /**
-   * Applies the `align-items` flex container property, affecting vertical `Col`
-   * alignment, for all screen sizes
+   * Applies the `align-items` flex container property to the row.
+   * This affects vertical `Col` alignment on all screen sizes.
```
Comment (jzempel) [inline packages/grid/src/elements/Row.tsx:24]: Yes, agreed on this use of `Col` 👍

**Review by jzempel — CHANGES_REQUESTED** [review]
For all use of `col`, I think it would be better to replace with `column`. We would say `Col` only if there was a particular reference to the component itself.

**Review by jzempel — COMMENTED** [review]
Any thoughts on https://github.com/zendeskgarden/react-components/pull/959#pullrequestreview-556692567?

**Comment by Mikaelia** [comment]
Ah yes, just updated!

**Review by jzempel — APPROVED** [review]

**Pull request merged**